examples/machinedeployment/kustomizeversions.yaml

```diff
@@ -40,18 +40,20 @@ spec:
             mkdir -p $CI_DIR
             CI_URL="gs://kubernetes-release-dev/ci/$CI_VERSION-bazel/bin/linux/amd64"
             declare -a PACKAGES_TO_TEST=("kubectl" "kubelet" "kubeadm")
-            PACKAGE_EXT="deb"
             declare -a CONTAINERS_TO_TEST=("kube-apiserver" "kube-controller-manager" "kube-proxy" "kube-scheduler")
             CONTAINER_EXT="tar"
 
             echo "* testing CI version $CI_VERSION"
 
             for CI_PACKAGE in "${PACKAGES_TO_TEST[@]}"; do
-            echo "* downloading package: $CI_URL/$CI_PACKAGE.$PACKAGE_EXT"
-            gsutil cp "$CI_URL/$CI_PACKAGE.$PACKAGE_EXT" "$CI_DIR/$CI_PACKAGE.$PACKAGE_EXT"
-            ${SUDO} dpkg -i "$CI_DIR/$CI_PACKAGE.$PACKAGE_EXT" || echo "* ignoring expected 'dpkg -i' result"
+            echo "* downloading binary: $CI_URL/$CI_PACKAGE"
+            gsutil cp "$CI_URL/$CI_PACKAGE" "$CI_DIR/$CI_PACKAGE"
+            chmod +x "$CI_DIR/$CI_PACKAGE"
+            mv "$CI_DIR/$CI_PACKAGE" "/usr/bin/$CI_PACKAGE"
             done
 
+            systemctl restart kubelet
```

Comment:
the kubelet has some dependencies:
https://github.com/kubernetes/release/blob/master/cmd/kubepkg/templates/latest/deb/kubelet/debian/control#L13

IMO not having these can cause issues:
iptables, kubernetes-cni, socat, mount.

and it's better to `apt-get install` fixed versions of them.

Reply:
So, currently the base image we are using has v1.16 packages already installed for `kubelet`, `kubeadm`, etc. Though, we should likely update that to v1.17.

The only time this would be a problem is if there is a new dependency or a new minimum requirement is added for the latest development release.